pyproject.toml

```diff
@@ -5,7 +5,7 @@ description = "Python library for Juju"
 readme = "docs/readme.rst"
 license = { file = "LICENSE" }
 maintainers = [{name = "Juju Ecosystem Engineering", email = "juju@lists.ubuntu.com"}]
-requires-python = ">=3.8"
+requires-python = ">=3.8.6"
```

Comment:
Yep. That's what they state.

We've got to support py 3.8, because we want a single virtualenv to be usable for both unit and integration tests.

Arguably the deployment base may not have been updated... 20.4 started with 3.8.2 or something, or maybe it has been updated...

I figure requiring a recent 3.8 in tests should be fine.

Reply:
Sweet, let's just see if this breaks anyone, low likelihood I'm guessing